setup/client_config.yaml

```
@@ -5,8 +5,12 @@ contexts:
     subsidies_object: 0xb606eb177899edc2130c93bf65985af7ec959a2755dc126c953755e59324209e
     exchange_objects: []
     wallet_config:
-      path: ~/.sui/sui_config/client.yaml
+      # Sui environment to use.
       active_env: mainnet
+      # Optional path to the wallet config file.
+      # path: ~/.sui/sui_config/client.yaml
+      # Optional override for the Sui address to use.
+      # active_address:
```

nit: This looks a little bit undone. Is there intended to be a value on the rhs of `active_address:`?
```suggestion
      # active_address: <0x...>
```